include/twin.h

```diff
@@ -808,6 +809,26 @@ void twin_path_arc(twin_path_t *path,
                    twin_angle_t start,
                    twin_angle_t extent);
 
+void twin_path_arc_ellipse(twin_path_t *path,
+                           bool large_arc,
+                           bool sweep,
+                           twin_fixed_t radius_x,
+                           twin_fixed_t radius_y,
+                           twin_fixed_t cur_x,
```

Comment:
You can use 
```c
twin_point_t cur_point;
```
Replace
```c
twin_fixed_t cur_x;
twin_fixed_t cur_y;
```

The advantage of doing this is that it directly separates the function of `radius_x` from the reading the code here, clearly indicating that it is an object representing a point.

Reply:
Yeah, I have also considered replacing `cur_x` and `cur_y` with `cur_point`. However, it might create an inconsistent style compared to other path functions like `twin_path_arc`, `twin_path_circle`, `twin_path_ellipse`.

Reply:
I got it.

Reply:
OK

I got it, close this review, please.